WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -566,7 +566,71 @@
         session[:locale] = :fr
       end
 
-      it "creates a results post" do
+      it "handles no 3x3x3 event" do
+        get :post_results, params: { id: competition }
+        post = assigns(:post)
+        expect(post.title).to eq "Results of #{competition.name}, in #{competition.cityName}, #{competition.countryId} posted"
+        expect(post.body).to eq "Results of the [#{competition.name}](#{competition_url(competition)}) are now available.\n\n"
+      end
+
+      context "winners announcement" do
+        let(:add_result) do
+          lambda do |pos, name|
```

Comment:
I believe this [can be a normal method](https://github.com/thewca/worldcubeassociation.org/blob/c2b39b5e984379ef9c7576be71f92607c57d4201/WcaOnRails/spec/lib/auxiliary_data_computation_spec.rb#L8-L17).